src/med_bench/estimation/tmle.py

```diff
@@ -0,0 +1,267 @@
+import numpy as np
+from sklearn.base import clone
+from sklearn.linear_model import LinearRegression
+
+from med_bench.estimation.base import Estimator
+from med_bench.nuisances.utils import (_get_regressor)
+
+ALPHA=10
+
+from med_bench.utils.decorators import fitted
+from med_bench.utils.utils import _get_interactions
```

Comment:
not sure we need interactions here, because we can use non-linear methods (and maybe some equivalent of the SuperLearner at some point), the place where interactions are key is for the coefficient product estimator